docs/using.rst

```diff
@@ -188,6 +190,24 @@ expected output is::
 .. _README: ../components/centos/centos-k8s-singlenode-setup/README.rst
 .. _Vagrantfile: ../components/centos/centos-k8s-singlenode-setup/Vagrantfile
 
+Vagrant 2 way folder sync
+=========================
+For basic usage please refer `Vagrant documentation. <https://www.virtualbox.org/manual/ch04.html#sharedfolders>`_
```

Comment:
This is the wrong link. This is VirtualBox specifc. What we are talking here are Vagrant Synced Folders - https://www.vagrantup.com/docs/synced-folders/